README.md

```diff
@@ -368,6 +384,8 @@ $builder->addStep(function ($item) {
 //[...]
 ```
 
+Note : you can prevent writing order for asynchronous operations if all steps are scaled at 1 factor.
```

Comment:
What do you mean by "prevent writing order" ?

Reply:
The wording was reverse ! I did change `prevent` to `ensure`. I mean that you can switch from synchronous to asynchronous an garantie the writing order when you scale all your steps at factor 1.